src/test/java/uk/ac/cam/cl/dtg/util/mappers/MapperTestUtils.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.cam.cl.dtg.util.mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class MapperTestUtils {
```

Comment:
This is actually a generic testing utils class that could potentially be reused for other use cases. Because of that, I would rename this to something like `Assertions`, and then move this file so that it sits in `uk.ac.cam.cl.dtg`.

Reply:
It looks like moving the class outside of the test folders prevents us from importing the Assertions used within the class. Changing that would presumably require making the testing library dependency universal rather than test-only and I'm doubtful that would be worth it.
I can rename the class and move it to the general test utils folder rather than the mapper subfolder though.
Let me know if you agree or not with my reasoning here? 🤔